apps/mobile/src/features/send-refactor/components/send-form-layout.tsx

```diff
@@ -0,0 +1,21 @@
+import { SpinnerIcon } from '@/components/spinner-icon';
+
+import { Box, BoxProps } from '@leather.io/ui/native';
+
+export function SendFormContainer(props: BoxProps) {
+  return <Box px="5" gap="3" flex={1} {...props} />;
+}
+
+export function SendFormFooter(props: BoxProps) {
+  return <Box flex={1} gap="3" justifyContent="flex-end" {...props} />;
+}
+
+export function SendFormLoadingSpinner() {
```

Comment:
Maybe this can be more generic as we need it in most places? I can refactor it when revisiting [LEA-1726](https://linear.app/leather-io/issue/LEA-1726/handle-balance-loading-state-elegantly)

Reply:
That would very nice. I intentionally kept it scoped to this flow for now because building a decent global page-level loading state will take a little more than this.